libecho/src/EchoRawConnectionImpl.h

```diff
@@ -13,28 +13,28 @@ class EchoRawConnectionImpl : public EchoRawConnection {
     /**
      * @brief @see EchoRawConnection::getBitEchoRawConnection
      */
-    static EchoRawConnectionImpl *getBitEchoRawConnection(int winSize, int receiveLoFreq, int receiveHiFreq,
-                                                          int sendLoFreq, int sendHiFreq, int leftLim, int rightLim);
+
+    static EchoRawConnectionImpl *getBitEchoRawConnection(int win_size, int send_freq, int recv_freq, int lim);
 
     /**
-     * @brief @see EchoRawConnection::send
+     * @brief @see EchoRawConnection::start
      */
-    void send(const std::vector<uint8_t> &buffer) override;
+    void sendStart() override;
```

Comment:
Why do names are not corresponding? (send in EchoRawConnection, sendStart in EchoRawConnectionImp and then send in ISender)

Reply:
because there would be 2 starts, and renaming them in ISender/Receiver is dumb